bookchain-miner/CMakeLists.txt

```diff
@@ -13,32 +13,36 @@ else()
 endif()
 
 
-# JSON library dependency
+# NLohmann JSON dependency
 set( JSON_BuildTests OFF CACHE INTERNAL "" )
 set( JSON_Install OFF CACHE INTERNAL "" )
 add_subdirectory( thirdparty/nlohmann_json )
 
 
+# Oat++ Dependency
+find_package( oatpp 1.0.0 REQUIRED )
```

Comment:
[Looks like](https://lgtm.com/projects/g/ErisMik/BookChain/logs/rev/pr-d2db788552cae16abbfdf24ef219eff46bf6a295/lang:cpp/stage:Build%20merge_8035873adf1efff8230ef070bdcb2c5547ac4e69) you may need to specify a Findoatpp.cmake so CMake can find an oat config.

Looks like it is failing on finding an oat config "oatppConfig.cmake" or "oatpp-config.cmake" in the build.

Reply:
I think the issue is not the the find-config is missing but that LGTM can't install the package and then cmake fails to find it (and gives that error message)

I need to add a LGTM conf file that will install the deps for us

Reply:
Alright, I've updated the CMAKE to pull in oatpp and build it if it can't find it.
This needed a newer version of CMAKE which is not in our docker build.
So now builds from scratch now take AGES but ¯\_(ツ)_/¯